Dangerfile

```diff
@@ -0,0 +1,34 @@
+# Sometimes it's a README fix, or something like that - which isn't relevant for
+# including in a project's CHANGELOG for example
+
+has_source_changes = !modified_files.grep(/Sources/).empty?
+has_test_changes = !modified_files.grep(/Tests/).empty?
+declared_trivial = pr_title.include? "#trivial" || !has_source_changes
+octo_client = env.request_source.client
+
+# Make a note about contributors not in the organization
+unless octo_client.organization_member?('VeniceX', pr_author)
+  message "@#{pr_author} is not a contributor yet, would you like to join VeniceX?"
+
+#  if modified_files.include?("*.gemspec")
+#    warn "External contributor has edited the Gemspec"
```

Comment:
Awesome! Actually I think this is a warning that should be presented when anyone changes `Package.swift`, not just external contributors. I think it's an important enough change to be easily visible.